public/docs/_examples/reactive-forms/ts/app/demo.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+<h1>Reactive Forms</h1>
+<h4><i>Pick a demo:</i>
+  <select [selectedIndex]="demo - 1" (change)="selectDemo($event.target.selectedIndex)">

```

Comment:
Interesting idea. I'm not using forms at all here and isn't particularly _form-y_. This is a pretty simple selection harness. Of course I'm open to a suggestion.